File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
import java.util.*;
class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] sList = s.split(" ");
        
        for (int i=0; i < sList.length; i++) {
            
            String a = sList[i];
            
            if(a.length() == 0) {
                answer += " ";
            } else {
                answer += a.substring(0, 1).toUpperCase();
                answer += a.substring(1, a.length()).toLowerCase();
                answer += " ";
            }
        }
        
        if (s.substring(s.length()-1, s.length()).equals(" ")) {
            return answer;
        }
        
        return answer.substring(0, answer.length()-1);
    }
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    answer = ''
    
    bol = True
    for i in s:
        if i == ' ':
            bol = True
            answer += i
        else:
            if bol:
                bol = False
                answer += i.upper()
            else:
                answer += i.lower()
    
    return answer